yesod-core/test/YesodCoreTest/Exceptions.hs

```diff
@@ -41,7 +41,7 @@ case500 :: IO ()
 case500 = runner $ do
     res <- request defaultRequest
     assertStatus 500 res
-    assertBody "FOOBAR" res
+    assertBodyContains "FOOBAR" res
```

Comment:
I'm still not following, sorry. There's a change here from `assertBody` to `assertBodyContains`, which is the difference between "is the body exactly X" and "does the body contain X." What's the relevance of `error` to that change?